<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget
{
font :'Montserrat';
}
QLabel
{
font-weight: bold;
}

#centralwidget{
background-color:rgb(243, 241, 241);
}

QPushButton::pressed
{
background-color: gray;
}

QLineEdit
{
border: 1px solid lightgray;
border-radius: 5px;
}

QTableView
{
border: 1px solid lightgray;
border-radius: 5px;
}

QComboBox
{
border: 1px solid lightgray;
border-radius: 5px;
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEdit_search_in_database">
      <property name="font">
       <font>
        <family>Montserrat Medium</family>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QTableView" name="tableView"/>
    </item>
    <item row="17" column="0">
     <widget class="QComboBox" name="comboBox_sqlQueries"/>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Montserrat ExtraBold</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Add data to Database</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="lineEdit_recordNumber">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Montserrat ExtraBold</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>View database table</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QComboBox" name="comboBox_editRecord"/>
    </item>
    <item row="13" column="1">
     <widget class="QPushButton" name="pushButton_add_new_data">
      <property name="text">
       <string>ADD DATA</string>
      </property>
     </widget>
    </item>
    <item row="18" column="0" colspan="2">
     <widget class="QLineEdit" name="lineEdit_customQuery">
      <property name="font">
       <font>
        <family>Montserrat Medium</family>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Record â„–:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="4">
     <widget class="QTableView" name="tableView_editDatabase">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Montserrat Medium</family>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QComboBox" name="comboBox_editDatabase"/>
    </item>
    <item row="13" column="0">
     <widget class="QTableView" name="tableView_addToDatabase">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>75</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>75</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="16" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Montserrat ExtraBold</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Sql Query</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QComboBox" name="comboBox_addToDatabase"/>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="pushButton_find_in_database">
      <property name="text">
       <string>Find</string>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QPushButton" name="pushButton_getRecord">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Get</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="10" column="0">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="18" column="2" colspan="2">
     <widget class="QPushButton" name="pushButton_custom_query_enter">
      <property name="text">
       <string>Enter</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Montserrat</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Edit database record</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QPushButton" name="pushButton_saveEdit">
      <property name="text">
       <string>Save edit</string>
      </property>
     </widget>
    </item>
    <item row="15" column="0">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="0">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="14" column="0">
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="Line" name="line_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
